openbas-api/src/main/java/io/openbas/scheduler/jobs/InjectsExecutionJob.java

```diff
@@ -108,16 +106,10 @@ private void executeExternal(ExecutableInject executableInject) {
               Injection source = executableInject.getInjection();
               Inject executingInject = null;
               InjectStatus injectRunningStatus = null;
-              DryInjectStatus dryInjectRunningStatus = null;
               if (source instanceof Inject) {
```

Comment:
question: Same as previous comment in regards to the instanceof check / change of signature to avoid using Injection interface